### PR TITLE
Bug fixes 0.8.x

### PR DESCRIPTION
**Pull request by tatarize**
(cherry picked from commit 26e0ec83a40cc4219eca75376496f4d6c36856e5)

### CODE REVIEW

File: meerk40t/core/svg_io.py [hunk 1]
```diff
@@ -1049,8 +1050,8 @@ def is_child(candidate, parent_node):
                         lock=_lock,
                     )
                     e_list.append(node)
-            except OSError:
-                pass
+            except OSError as e:
+                raise e
```
Comment (Sophist-UK) [inline meerk40t/core/svg_io.py:1054]: Isn't this the same as not doing an except?

Comment (tatarize) [reply meerk40t/core/svg_io.py:1054]: Yes. This like was debug. I needed to see what e actually said as part of the debugging.

Comment (Sophist-UK) [reply meerk40t/core/svg_io.py:1054]: No - my point is that trapping and then raising the same exception is EXACTLY the same as not trapping it at all.

Comment (tatarize) [reply meerk40t/core/svg_io.py:1054]: I did that to look at the current value of `e` which let me see what the thrown error was. Which is slightly different. But, yes, you are correct. I will remove that change.

Comment (tatarize) [reply meerk40t/core/svg_io.py]: This and all changes to svg_io were reverted.

**Pull request merged**